lpcompiler.pas

```diff
@@ -2740,6 +2787,44 @@ function TLapeCompiler.ParseExpression(ReturnOn: EParserTokenSet = []; FirstNext
             Dec(InExpr);
           end;
 
+        tk_sym_BracketOpen:
+          if (_LastNode = _Var) then
+          begin
+            PopOpStack(op_Invoke);
```

Comment:
Sure, `op_Invoke` is `op_Index` tho, which is why I used it, as we are in fact invoking it, just felt it gave a clearer message to use `op_Invoke`.